coral/src/app/pages/topics/index.test.tsx

```diff
@@ -58,6 +62,47 @@ describe("Topics", () => {
       expect(headline).toBeVisible();
     });
 
+    it("renders 'Request A New Topic' primary action in heading", async () => {
+      const headline = screen.getByRole("button", {
+        name: "Request A New Topic",
+      });
+
+      expect(headline).toBeVisible();
+    });
+
+    it("renders 'Request A New Topic' primary action in heading", async () => {
+      const primaryAction = screen.getByRole("button", {
+        name: "Request A New Topic",
+      });
+
+      expect(primaryAction).toBeVisible();
+    });
+
+    it("navigates to '/requestTopics' on clicking the primary action in heading", async () => {
+      // This TS disabling is because of "The operand of a 'delete' operator must be optional." TS error.
+      // But this delete is necessary to correctly mock window.location
+      // Without it, we get "Error: Not implemented: navigation (except hash changes)" in JSDOM
+      // Sources:
+      //  https://stackoverflow.com/questions/54090231/how-to-fix-error-not-implemented-navigation-except-hash-changes
+      //  https://remarkablemark.org/blog/2021/04/14/jest-mock-window-location-href/
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      delete window.location;
+      window.location = {
+        href: "/topics",
+      } as Location;
+
+      const primaryAction = screen.getByRole("button", {
+        name: "Request A New Topic",
+      });
+
+      fireEvent.click(primaryAction);
```

Comment:
Small note: We're using `userEvent` instead of `fireEvent`. It's a (more user oriented and convenient) wrapper around `fireEvent` so does not make a difference in the test here, but I would replace it to be more consistent! 

Probably something that could also be part of linting later 🤔

Reply:
Oops yes, good point! It's actually in the docs, should have remembered ^^'